### DISCUSSION TITLE
Open Visualization Collaborator Summit in Madrid, Spain | September 22-23, 2022

### DISCUSSION THREAD

**Discussion by Pessimistress** [visgl/deck.gl#7217] [Announcement]
Come together with the vis.gl/deck.gl/kepler.gl community for two days of talks, demos, and workshops!

Learn more at https://deck.gl/events/madrid-summit-2022/

**Comment by davidcalhoun** [comment]
Great!  Will these talks be recorded?

**Reply by Pessimistress** [reply]
Still working on the logistics, but I believe they will at least be streamed during the event.
